site/src/pages/DeploySettingsPage/NotificationsPage/NotificationsPage.tsx

```diff
@@ -187,6 +201,13 @@ const EventsView: FC<EventsViewProps> = ({
           Webhook method is enabled, but the endpoint is not configured.
         </Alert>
       )}
+
+      {isUsingSmpt && !smtpConfig && (
```

Comment:
I don't think SMTP config can be blank.

Reply:
From the types, it could be tho, but you shared this message a few minutes ago:

> We can't assume SMTP will always be configured. We should have the same check here.

Reply:
Done!